app/code/Magento/Bundle/Test/Mftf/Test/AdminDisappearErrorValidateMessageFieldTest.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<!-- Test XML Example -->
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminDisappearErrorValidateMessageFieldTest">
```

Comment:
Please name test  more descriptiive
```
AdminBundleProductPriceValidationErrorDisappearedAfterSwitchToDynamicPriceTest
```
Dont forget rename file and change Test Path for corresponding zephyr test MC-40309